python/pyarrow/tests/test_flight.py

```diff
@@ -129,6 +129,15 @@ def __init__(self, location=None, **kwargs):
             b'dicts': simple_dicts_table,
         }
 
+    def list_flights(self, context, criteria):
+        if criteria:
```

Comment:
Should you check a well-defined criteria value here to make the test more interesting?

Reply:
Good idea, I've updated the test.